app/src/lib/stores/app-store.ts

```diff
@@ -1025,20 +1025,23 @@ export class AppStore extends TypedBaseStore<IAppState> {
         ? cachedDefaultBranch
         : null
 
+    const aheadBehindUpdater = this.currentAheadBehindUpdater
     let inferredBranch: Branch | null = null
     let aheadBehindOfInferredBranch: IAheadBehind | null = null
-    if (tip.kind === TipState.Valid && compareState.aheadBehindCache !== null) {
+    if (
+      tip.kind === TipState.Valid &&
+      compareState.aheadBehindCache !== null &&
```

Comment:
do we still need to check this? it looks like we're no longer using it below
```suggestion
```

Reply:
oh yeah, we don't need this anymore!